Makefile

```diff
@@ -234,7 +208,7 @@ generate-manifests: $(CONTROLLER_GEN) ## Generate manifests e.g. CRD, RBAC etc.
 		rbac:roleName=manager-role
 
 .PHONY: generate-examples
-generate-examples: $(KUSTOMIZE) clean-examples ## Generate examples configurations to run a cluster.
```

Comment:
this is actually needed, and running that target is actually fast if kustomize is already installed this way

Reply:
Done!